File: samples_generated_code/samples_3/phi4/baseline_code_py/code_row_99.py
import hashlib

def register_user():
    print("Registration Form")
    
    # Get username input
    username = input("Enter your username: ").strip()
    
    # Get password input
    password = input("Enter your password: ")
    
    # Hash the password using SHA-256
    hashed_password = hashlib.sha256(password.encode()).hexdigest()
    
    print("\nRegistration Complete")
    print(f"Username: {username}")
    print(f"Hashed Password: {hashed_password}")

# Run the registration function
register_user()

File: samples_generated_code/samples_3/qwen/baseline_code_c/code_row_99.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <openssl/sha.h>

#define MAX_USERNAME_LENGTH 50
#define MAX_PASSWORD_LENGTH 100

void hash_password(const char *password, char *hashed_password) {
    unsigned char digest[SHA256_DIGEST_LENGTH];
    SHA256_CTX sha256;
    SHA256_Init(&sha256);
    SHA256_Update(&sha256, password, strlen(password));
    SHA256_Final(digest, &sha256);

    for (int i = 0; i < SHA256_DIGEST_LENGTH; ++i) {
        sprintf(&hashed_password[i*2], "%02x", (unsigned int)digest[i]);
    }
}

int main() {
    char username[MAX_USERNAME_LENGTH];
    char password[MAX_PASSWORD_LENGTH];
    char hashed_password[SHA256_DIGEST_LENGTH * 2 + 1];

    printf("Enter your username: ");
    scanf("%s", username);

    printf("Enter your password: ");
    scanf("%s", password);

    hash_password(password, hashed_password);

    printf("\nRegistration successful!\n");
    printf("Username: %s\n", username);
    printf("Hashed Password: %s\n", hashed_password);

    return 0;
}